### ISSUE TITLE
Scenes are configured but do not show up in Express

### ISSUE THREAD

**Issue by sahafeez** [kuestess/homebridge-platform-insteonlocal#210]
So it is possible I am misunderstanding something here. With my old HUB I could see the scenes and their members links in the hub information/scenes section. I moved everything I had to the USB2448A7 fresh using the old houselinc software. I also created a bunch of scenes in that process. In the express server the scenes exist as you see the responder / controller groups, but they have no members except the hub as a controller. Going over to the device itself and pulling links then looking at the scenes there do show the memberships.

**Comment by kuestess** [comment]
@sahafeez You are correct - assuming that HouseLinc created the scenes/linking correctly you should see them there similar to what you did with the hub.  Wondering whether this is related to your other issue with the 2448A7.  Have you tried linking a device and creating a scene using the ui to see if that works?

**Comment by kuestess** [comment]
@sahafeez I did a little searching on the internets and it seems like there are some differences in the protocol for the 2448A7, but it doesn't necessarily explain this behavior or the other issues you are seeing.  Can you try out linking a device via the ui and only adding that device to the config to see if the error issues still pop up.

**Comment by sahafeez** [comment]
@kuestess So I created a scene in express and it worked. I then took a look at the db under houselinc. The scene created in express does not show. 

I am going to delete all the scenes from houselinc and try to recreate them in express. The one issue I am facing is I have a 8 button remote that I want to link the scene to, and that works in houselinc but I cannot seem to make it work (link to the correct button) in express.
